students/hanjoon/users/views.py

```diff
@@ -32,4 +32,23 @@ def post(self, request):
             )
             return JsonResponse({"message":"SUCCESS"}, status = 201)
         except KeyError:
-            return JsonResponse({"MESSAGE":"KEY_ERROR"}, status = 400)
\ No newline at end of file
+            return JsonResponse({"MESSAGE":"KEY_ERROR"}, status = 400)
+
+class LoginView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        password = data["password"]
+
+        user = User.objects.get(email=data["email"])
+
+        try:
+            if password == user.password:
+                return JsonResponse({"MESSAGE":"SUCCESS"}, status = 200)
+            else:
+                return JsonResponse({"MESSAGE":"INVALID_INFO"}, status = 400)
```

Comment:
if 조건에 부합한 경우 return 으로 로직이 종료됩니다.
즉, 부합하지 않은 경우 바로 다음 로직으로 진행된다는 말이 됩니다!
꼭 필요한 경우가 아니라면 else, elif 를 덜 사용하는 것이 파이썬스럽습니다!
```suggestion
        try:
            if password == user.password:
                return JsonResponse({"MESSAGE":"SUCCESS"}, status = 200)
            return JsonResponse({"MESSAGE":"INVALID_INFO"}, status = 400)
```